### DISCUSSION TITLE
Welcome to mmbot Discussions!

### DISCUSSION THREAD

**Discussion by ondra-novak** [ondra-novak/mmbot#42] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by w2vy** [comment]
How is the project going? Is there a discord? The invite in the code has expired...

**Reply by ondra-novak** [reply]
This should work: https://discord.com/invite/zPrZRNgvWD

Currently I am in a phase of preparing a new project which should replace this project in a future. The main goal is to rewrite the main trading cycle and overall to change the layout of the code. For example, I wish to have an option to split instance to multiple instances with ability to be installed at server at location near to the exchange and still to have ability to manage everything from one place.
